src/pages/Signup/Signup.js

```diff
@@ -0,0 +1,310 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { RiArrowRightSLine } from 'react-icons/ri';
+import './Signup.scss';
+
+class Signup extends Component {
+  state = {
+    form: {
+      account: '',
+      password: '',
+      pwCheck: '',
+      name: '',
+      year: '',
+      month: '',
+      day: '',
+      email: '',
+      emailDot: '',
+      phoneNum1: '',
+      phoneNum2: '',
+      phoneNum3: '',
+    },
+    phoneNum: '',
+    date: '',
+    email: '',
+  };
+
+  handleOnchange = e => {
+    const { name, value } = e.target;
+    const { form } = this.state;
+    this.setState(
+      {
+        ...this.state,
+        form: {
+          ...this.state.form,
+          [name]: value,
+        },
```

Comment:
class 형 컴포넌트에서 setState 는 병합을 합니다. 그렇기 때문에, 
```suggestion
{
  form: {
    ...this.state.form,
    [name]: value,
  }
}
```
만 해주셔도 됩니다!